scripts/psiphon_install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
No, it's not intented, my fault.
Note for this comment and the rest, that i just added this script so it's easy for others to check the test, but i didn't want it to be merged and i should not add it to this PR, as i'd like to talk first with @anadahz on how the installation script should be done.